Installation.bat

```diff
@@ -0,0 +1,17 @@
+haxelib install lime
+haxelib install openfl
+haxelib install flixel
+haxelib run lime setup flixel
+haxelib run lime setup
+haxelib install flixel-tools
+haxelib run flixel-tools setup
+haxelib update flixel
+haxelib install flixel
+haxelib install flixel-addons
+haxelib install flixel-ui
+haxelib install hscript
+haxelib install newgrounds
+haxelib git polymod https://github.com/larsiusprime/polymod.git
+haxelib git discord_rpc https://github.com/Aidan63/linc_discord-rpc
+haxelib git flixel-addons https://github.com/HaxeFlixel/flixel-addons
+haxelib git linc_luajit https://github.com/AndreiRudenko/linc_luajit
```

Comment:
Nothing is wrapped inside of a function, this isn't how batch file works.

Reply:
> This is not how batchfile works. This doesn't do anything at all.

I literally used it to install the dependencies. It does work like that

Reply:
this is working for me actually, now thats what a batchfile works